### PR TITLE
Adding cron job to delete old partitions from vertica

### PR DESCRIPTION
**Pull request by mhoppal**

### CODE REVIEW

File: tasks/vertica.yaml [hunk 1]
```diff
@@ -36,5 +36,7 @@
     - mon_users.sql
   when: sql_file_status|changed or sql_template_status|changed
 
+- name: Create cron job to clean up old partitions
+  template: src=vertica-cleanup.j2 dest=/etc/cron.daily/vertica-cleanup mode=755
```
Comment (tkuhlman) [inline tasks/vertica.yaml]: These permissions are too open for a file that contains the vertica dbadmin password. I would make it 700 or 750.

**Comment by tkuhlman** [comment]
Cron will typically send an email out to the running user regarding any script with output. What does the output look like? It may be best to have it send everything to a log file, maybe one for the month, then silence any console output.

Also I'm curious how long does it take to run?

**Comment by mhoppal** [comment]
Right now it takes a second or two to run but given Vertica has only been running for a couple of days so probably not the best test. The output is just "Partition dropped" repeated for each partition dropped nothing useful to the user in my opinion. What do you think? I mean we can add on the date for when it was dropped to the output and send that to a log under /opt/vertica/var. I will change the permission problem I just matched the rest of the cron scripts in that directory but what you said makes sense.

**Comment by tkuhlman** [comment]
Actually just partition dropped isn't bad if it goes out in a standard cron email.

**Comment by roland-hochmuth** [comment]
ok

**Comment by mhoppal** [comment]
Worked when running as a cron job. Mail that was sent to user was:

Date: Tue,  2 Jun 2015 14:36:01 +0000 (UTC)

/etc/cron.hourly/vertica-cleanup:
 Partition dropped

**Pull request merged**